src/wallet/rpcwallet.cpp

```diff
@@ -2909,6 +2909,128 @@ UniValue z_listunspent(const UniValue& params, bool fHelp)
     return results;
 }
 
+UniValue z_listutxos(const UniValue& params, bool fHelp)
+{
+
+    if (!EnsureWalletIsAvailable(fHelp))
+        return NullUniValue;
+
+    if (fHelp || params.size() < 1 || params.size() > 5)
+        throw runtime_error(
+            "z_listutxos account ( minconf maxconf asOfHeight )\n"
+            "\nReturns the unspent transparent outputs for the account."
+            "\nArguments:\n"
+            "1. account      (numeric) The account number.\n"
+            "2. minconf      (numeric, optional, default=1) Only include transactions confirmed at least this many times.\n"
+            "3. maxconf      (numeric, optional, default=9999999) The maximum confirmations to filter\n"
```

Comment:
I don't personally see a reason for this argument to exist, but whoever added it to `listunspent` probably knew more than I do.